### PR TITLE
Copyedited Documentation

### PR DESCRIPTION
**Pull request by Pow3r5**
## Overview: What does this pull request change?

Reviewed tutorials\configurations.rst. Edited simple mistakes such as Manim not being capitalized and commas.

## Motivation and Explanation: Why and how do your changes improve the library?

My motivation was to start getting involved in the open-source community. This improves the library as it increases readability and professionalism.

## Reviewer Checklist
- [ ] The PR title is descriptive enough for the changelog, and the PR is labeled correctly
- [ ] If applicable: newly added non-private functions and classes have a docstring including a short summary and a PARAMETERS section
- [ ] If applicable: newly added functions and classes are tested

### CODE REVIEW

**Comment by tameTNT** [comment]
Would it be possible for you to edit your comment to remove the `closes` keyword so that #1401 remains open after your PR has been approved?

**Comment by Pow3r5** [comment]
> Would it be possible for you to edit your comment to remove the `closes` keyword so that #1401 remains open after your PR has been approved?

Yep will do sorry about that

File: docs/source/tutorials/configuration.rst [hunk 1]
```diff
@@ -188,8 +188,8 @@ is in the same directory as <file.py>,
 
 Since config files are meant to replace CLI flags, all CLI flags can be set via
 a config file.  Moreover, any config option can be set via a config file,
-whether or not it has an associated CLI flag.  For a list of all CLI flags and
-all config options, see the bottom of this document.
+whether or not it has an associated CLI flag.  See the bottom of this document
+for a list of all CLI flags and all config options.
```
Comment (tameTNT) [inline docs/source/tutorials/configuration.rst:192]:
```suggestion
for a list of all CLI flags and config options.
```
Second `all` is unnecessary I think.

**Review by tameTNT — APPROVED** [review]
These changes all seem good to me. As you say, mainly just capitalising `manim` and improving clarity with commas.

File: docs/source/tutorials/configuration.rst [hunk 2]
```diff
@@ -8,20 +8,20 @@ can be configured programmatically via `the ManimConfig class`_, at the time
...
-The most common, simplest and recommended way to set the configure Manim is
+The most common, simplest, and recommended way to set the configure Manim is
 via the command-line interface (CLI), which is described directly below.
 
 Command-line arguments
 **********************
 
-By far the most commonly used command in the CLI is the ``render`` command,
-which is used to render scene(s) to an output file.
+By far, the most commonly used command in the CLI is the ``render`` command,
```
Comment (Darylgolden) [inline docs/source/tutorials/configuration.rst:17]:
```suggestion
By far the most commonly used command in the CLI is the ``render`` command,
```

I believe this is incorrect.

File: docs/source/tutorials/configuration.rst [hunk 3]
```diff
@@ -8,20 +8,20 @@ can be configured programmatically via `the ManimConfig class`_, at the time
...
+which is used to render the scene(s) to an output file.
```
Comment (Darylgolden) [inline docs/source/tutorials/configuration.rst:18]:
```suggestion
which is used to render scene(s) to an output file.
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]

File: docs/source/tutorials/configuration.rst [hunk 4]
```diff
@@ -8,20 +8,20 @@ can be configured programmatically via `the ManimConfig class`_, at the time
 of command invocation via `command-line arguments`_, or at the time the library
 is first imported via `the config files`_.
 
-The most common, simplest and recommended way to set the configure Manim is
+The most common, simplest, and recommended way to set the configure Manim is
```
Comment (behackl) [inline docs/source/tutorials/configuration.rst:11]:
```suggestion
The most common, simplest, and recommended way to configure Manim is
```

**Review by behackl — APPROVED** [review]
Thank you for your efforts! I've added one more suggestion myself and applied all of the suggestions; this is good to go now. :-)

**Pull request merged**